Nil pointer dereference: ClientTx.actInviteResend

Hi @emiago! First off I just wanted to say great work on this project! Its great to finally have a solid SIP library for Go. Great work!! 

Similarly to #238  we are running into a nil pointer dereference panic in `ClientTx.actInviteResend` under high load when responses are delayed from server. 

```go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x4b654e]

goroutine 430 [running]:
time.(*Timer).Reset(0x115b1eb3adf0?, 0x425905?)
  /usr/local/go/src/time/sleep.go:172 +0xe
github.com/emiago/sipgo/sip.(*ClientTx).actInviteResend(0x115b1ef12680)
  /go/pkg/mod/github.com/emiago/sipgo@v1.2.1/sip/transaction_client_tx_fsm.go:202 +0x55
github.com/emiago/sipgo/sip.(*ClientTx).inviteStateCalling(0x115b1ef12680, 0x1893900?)
  /go/pkg/mod/github.com/emiago/sipgo@v1.2.1/sip/transaction_client_tx_fsm.go:39 +0x357
github.com/emiago/sipgo/sip.(*baseTx).spinFsmUnsafe(0x115b1ef12680, 0x115b1e8517d0?)
  /go/pkg/mod/github.com/emiago/sipgo@v1.2.1/sip/transaction.go:244 +0x30
github.com/emiago/sipgo/sip.(*baseTx).spinFsm(0x115b1ef12680, 0x0?)
  /go/pkg/mod/github.com/emiago/sipgo@v1.2.1/sip/transaction.go:251 +0x57
github.com/emiago/sipgo/sip.(*ClientTx).Init.func1()
  /go/pkg/mod/github.com/emiago/sipgo@v1.2.1/sip/transaction_client_tx.go:60 +0x1e
created by time.goFunc
  /usr/local/go/src/time/sleep.go:215 +0x2d
```

Nil check resolves the issue here as well in our case. 

```go
func (tx *ClientTx) actInviteResend() fsmInput {
	tx.mu.Lock()

	tx.timer_a_time *= 2

	if tx.timer_a != nil {
		tx.timer_a.Reset(tx.timer_a_time)
	}

	tx.mu.Unlock()

	tx.resend()

	return FsmInputNone
}
```

--- a/sip/transaction_client_tx_fsm.go
+++ b/sip/transaction_client_tx_fsm.go
@@ -199,7 +199,10 @@ func (tx *ClientTx) actInviteResend() fsmInput {
 	tx.mu.Lock()
 
 	tx.timer_a_time *= 2
-	tx.timer_a.Reset(tx.timer_a_time)
+
+	if tx.timer_a != nil {
+		tx.timer_a.Reset(tx.timer_a_time)
+	}
 
 	tx.mu.Unlock()
 

